### ISSUE TITLE
Can't use both the --raw-output and --ascii-output switches

### ISSUE THREAD

**Issue by eelcohn** [stedolan/jq#1587]
If both the --raw-output and the --ascii-output command line switches are used, the --ascii-output switch is ignored:
```
$ echo "{\"key1\":\"\\ud83d\\ude00\"}" | jq --raw-output '.key1'
😀
$ echo "{\"key1\":\"\\ud83d\\ude00\"}" | jq --ascii-output '.key1'
"\ud83d\ude00"
$ echo "{\"key1\":\"\\ud83d\\ude00\"}" | jq --raw-output --ascii-output '.key1'
😀
```
What I'd expect is that in the last example the output would be:
```
$ echo "{\"key1\":\"\\ud83d\\ude00\"}" | jq --raw-output --ascii-output '.key1'
\ud83d\ude00
```

**Comment by rain-1** [comment]
https://github.com/stedolan/jq/pull/1611

**Comment by nicowilliams** [comment]
Closed with bf88c73c20ff1642aa2566ae67df18ca5b4b63ab via #1587 by @rain-1.  Thanks!
